### ISSUE TITLE
asd find crashrecovery needs rework

### ISSUE THREAD

**Issue by tsonntig** [graysky2/anything-sync-daemon#56]
One Problem with asd finding wrong crashrecoverys :

```
Anything-sync-daemon v5.85 on Arch Linux

 Systemd service is currently active.
 Systemd resync service is currently active.
 Overlayfs v23 is currently active.


Asd will manage the following per /run/asd.conf settings:

 owner/group id:     thomas/100
 target to manage:   /home/thomas/.cache
 sync target:        /home/thomas/.cache-backup_asd
 tmpfs target:       /tmp/asd-thomas/home/thomas/.cache
 dir size:           1,3G
 overlayfs size:     17M
 recovery dirs:      1 <- delete with the c option
  dir path/size:     /home/thomas/.mozilla/firefox/thomas_firefox_standart-backup-crashrecovery-20190722_020242 (346M)
```

The second Problem is that the find *crashrecovery* command takes a long time on startup.

My suggested Solution:
change 
`find "${BACKUP%/*}" -type d -name '*crashrecovery*' 2>/dev/null|sort -r`
to 
`find "${BACKUP%/*}" -maxdepth 1 -type d -name '*crashrecovery*' 2>/dev/null|sort -r`

**Comment by magnetophon** [comment]
Just ran into this as well.

**Comment by manorit2001** [comment]
Fixed in `v5.86`
